swc/extensions.bzl

```diff
@@ -9,11 +9,20 @@ swc_toolchain = tag_class(attrs = {
     # "swc_version_from": attr.string(doc = "Location of package.json which may have a version for @swc/core."),
 })
 
+default_repository = "swc"
+
 def _toolchain_extension(module_ctx):
     registrations = {}
     for mod in module_ctx.modules:
         for toolchain in mod.tags.toolchain:
+            if toolchain.name != default_repository and not mod.is_root:
+                fail("Only the root module may provide a name for the {} toolchain.".format(toolchain.name))
+
             if toolchain.name in registrations.keys():
+                if toolchain.name == default_repository:
```

Comment:
Nice, we should drive by the other rulesets since we probably copied the same mistake